packages/pf4-component-mapper/src/files/dual-list-select.js

```diff
@@ -306,7 +332,8 @@ DualList.defaultProps = {
   filterValueText: 'Remove your filter to see all selected',
   options: [],
   allToLeft: true,
-  allToRight: true
+  allToRight: true,
+  renderStatus: null
```

Comment:
Can you remove this default prop? It's not necessary.

Reply:
Done :smile_cat: